### PR TITLE
53 create login form

### PR DESCRIPTION
**Pull request by elios11**

### CODE REVIEW

File: src/components/LoginForm/LoginForm.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    return (
+        <form className={styles.form}>
+            <input
+                type="email"
+                name="email"
+                placeholder="Email"
+                autoComplete="email"
+                id="email"
+                className={styles['form__input']}
+                method="post"
```
Comment (fgomensoro) [inline src/components/LoginForm/LoginForm.jsx]: i think method atribute goes in form tag

File: src/helper-styles/colors.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
+:root {
```
Comment (fgomensoro) [inline src/helper-styles/colors.css]: maybe this folder can go inside utils

**Review by fgomensoro — APPROVED** [review]

File: src/components/LoginForm/LoginForm.jsx (hunk 1 above)
Comment (elios11) [reply src/components/LoginForm/LoginForm.jsx]: lmao, going

**Comment by fgomensoro** [comment]
the method attribute in form its not necessary in react because the information is handled commonly with a handleSubmit.

const handleSubmit = (e) => {
    e.preventDefault();
    // Handle form submission logic here
    console.log('Form submitted with data:', formData);
  };

  return (
    <form onSubmit={handleSubmit}>
      <label>
        Name:
        <input type="text" name="name" value={formData.name} onChange={handleInputChange} />
      </label>
      <label>
        Email:
        <input type="email" name="email" value={formData.email} onChange={handleInputChange} />
      </label>
      <button type="submit">Submit</button>
    </form>
  );

**Review by fgomensoro — APPROVED** [review]

**Comment by elios11** [comment]
Do you think I should be adding that functionality inside of the same component?

**Comment by fgomensoro** [comment]
> Do you think I should be adding that functionality inside of the same component?

if you want you can look up for the react hook form library validation. Its the one mostly used. It handles the result, errors, and validations. Also in combination with yup library wich adds a schema object to the formData.

[https://react-hook-form.com/](url)
[https://www.npmjs.com/package/yup](url)

Although I think it may be more organized to do it in another branch.

File: src/helper-styles/colors.css (hunk 2 above)
Comment (elios11) [reply src/helper-styles/colors.css]: done!

**Review by valselviz — APPROVED** [review]

**Pull request merged**